lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -55,7 +28,7 @@ def to_representation(self, course_overview):
             start_type = "empty"
             start_display = None
```

Comment:
This will benefit from the same refactor as the work I did on the course api serializer.  start_type and start_display should probably be properties on the `CourseOverview` object (and other representations of courses?)